governance-charter.md

```diff
@@ -159,19 +164,18 @@ to the respective positions.
 
 ### Maximal representation
 
-To encourage diversity there will be a maximum of one-third representation on
-the Governance Committee from any one company at any time. If the outcomes of
-an election result in greater than 1/3 representation (or maximum of two,
-whichever is greater), the lowest vote getters from any particular company will
-be removed until representation on the committee is less than one-third.
-
-If percentages shift because of job changes, acquisitions, or other events,
-sufficient members of the committee must resign until max one-third
-representation is achieved. If it is impossible to find sufficient members to
-resign, the entire company’s representation will be removed and new special
-elections held. In the event of a question of company membership (for example
-evaluating independence of corporate subsidiaries) a majority of all
-non-involved Governance Committee members will decide. 
+To encourage diversity there will be a maximum of two Governance Committee
+members from any one company at any time. If the outcomes of an election result
+in more than two candidates from the same company being elected, the lowest vote
+getters from any particular company will be removed.
```

Comment:
What about the case that one (or two) employees from one company are already members of the GC, and one (or more) additional employees of that same company are elected? In the case of 2 existing members, are additional employees even _able_ to be elected? In the case of 1 existing member, and 2 additional being elected, is the same logic applied (the lowest vote getters will be removed)?

Reply:
I think the first sentence summarizes the spirit of the rule:

> To encourage diversity there will be a maximum of two Governance Committee
members from any one company at any time.

If the are two people already at the board, no further candidates can be elected from the same company without one of those resigning.

Reply:
> To encourage diversity there will be a maximum of two Governance Committee
members from any one company at any time. If the outcomes of an election result
in more than two candidates from the same company being at the GC, the lowest vote
getters from any particular company will be removed. 

Sorry, for being nit-picking, but while this wording is improved around the fact that there may be GC members already, it's not entirely clear what "the lowest vote getters from any particular company will be removed" means: is this only comparing votes of the current election, or also from previous elections? E.g. if a GC member got 100 votes in year 1 and then two new GC members are elected with 101 and 102 votes in year 2, is the member with 100 votes in year1 removed or the one with 101 votes from year 2? In the case of the existing GC member from year1 being removed (or resigning deliberately), will there be another special election or will one additional person be added to the GC in this election (and then only serve one year?)?

Reply:
A different proposal may be the following:

* Name GC members elected in odd years "Cohort 1" and those elected in even years "Cohort 2".
* In each cohort there must only be a maximum of one GC members from any one company at any time. 

In a case of an election there will be no more conflict with "existing" and "newly elected" members. In the case of a company change the existing rules may apply.

Reply:
I agree with @svrnm it should be clear that only the current election is affected. We should also think about what happens if there is a merge/acquisition that changes the GC composition outside of an election cycle. Do we let it ride until the elections naturally fix the issue, do we allow the employees of the company to decide amongst themselves who should step down and have a special election, or do we drop the lowest voted member of the most recent cohort?

Reply:
Keeping track of old election results and assuming they are still accurate by the time a decision is made doesn't sound right to me. I'm also not very fond of having a max per cohort.

I believe the effective rule right now is that new members cannot be elected if two members from the same company are on the board already. I changed the wording on this doc to reflect that, but we can certainly discuss other alternatives if that's not the understanding of the majority.

Reply:
> it should be clear that only the current election is affected.

That's why I suggested the solution of having 1 person per company per election. It _might_ make things easier

> We should also think about what happens if there is a merge/acquisition that changes the GC composition outside of an election cycle. Do we let it ride until the elections naturally fix the issue, do we allow the employees of the company to decide amongst themselves who should step down and have a special election, or do we drop the lowest voted member of the most recent cohort?

There is a section about the below the election ("_If representation changes because of job changes, acquisitions, or other events, sufficient members of the committee must resign within 28 days until the maximum representation is respected._"). So it is left to the affected GC members to figure out a solution themselves.t.

Reply:
I wrote & send my last comment while yours arrived. I'll leave it for completeness but no need to respond.

> I believe the effective rule right now is that new members cannot be elected if two members from the same company are on the board already. I changed the wording on this doc to reflect that, but we can certainly discuss other alternatives if that's not the understanding of the majority.

While not _perfect_ this is a good solution (also compared to the other ones), although there are 2 clarifications that need to be made:

* It should be explicit, that those 2 existing members should not be up for re-election
* If there is one existing member of a company, and more than one (2+) get elected the rule with the "most votes" should still apply.

I can try to make some suggestions